### PR TITLE
update codeowners for new SOP

### PR DESCRIPTION
**Pull request by deandreJones**
updating code owners for integration pull request

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,4 +6,28 @@
...
+
+
+# Require tech lead review for changes to the following files
+.golangci.yml @transcom/codeowners
+.pre-commit-config.yaml @transcom/codeowners
+.eslintrc.js @transcom/codeowners
+/eslint-plugin-ato/ @transcom/codeowners
+/pkg/ato-linter/ @transcom/codeowners
+/scripts/ @transcom/codeowners
+/.circleci/config.yml @transcom/codeowners
```
Comment (cameroncaci) [inline .github/CODEOWNERS]: Recommend adding `Dockerfile*`, `Brewfile*`, `go.mod`

**Review by danieljordan-caci — APPROVED** [review]
LGTM!

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -6,4 +6,28 @@
 # owners will be requested for a review.
 # Add language specific code owners if it becomes relevant
 
+# database team
 /migrations/ @transcom/cacidatabaseteam
+/pkg/assets/sql_scripts/ @transcom/cacidatabaseteam
+
+# backend team
+/pkg/ @transcom/cacibackendteam
+/swagger-def/ @transcom/cacibackendteam
```
Comment (cameroncaci) [inline .github/CODEOWNERS:15]: Recommend adding `go.mod`

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -6,4 +6,31 @@
...
+*.go @transcom/cacibackendteam
+
+
+# frontend team
+/src/ @transcom/cacifrontendteam
+/playwright/tests/ @transcom/cacifrontendteam
```
Comment (cameroncaci) [inline .github/CODEOWNERS:21]: Recommend adding `package.json`

Comment (deandreJones) [reply .github/CODEOWNERS:21]: good call

Comment (deandreJones) [reply .github/CODEOWNERS:21]: ill put that under codeowner

**Review by cameroncaci — APPROVED** [review]
🚀      🌔

**Review by traskowskycaci — APPROVED** [review]
No critiques!

**Pull request merged**